### ISSUE TITLE
Mention possible follow-ups

### ISSUE THREAD

**Issue by gsmet** [quarkusio/quarkus-backports#232]
We won't find the ideal solution but I think we should at least experiment if we could hint at possible follow-ups in the UI.

A follow up would be a PR mentioning the PR at hand. And we would display them as `Possible follow-ups` in the UI just below the list of commits. I think a simple list on a single line with `#xxxx` title, `#yyyy` title would be good enough.

There are a few pointers here: https://github.com/orgs/community/discussions/24367 .

Now we might need to be a bit more efficient that doing it per issue as we don't want to be rate limited and we might have ~100 PRs to backport for a given version.

**Comment by gsmet** [comment]
@jmartisk in case you want to get familiar with the tooling. Definitely not high priority but if you want to get some GraphQL fun... :)

**Comment by jmartisk** [comment]
Sounds like this may even be work for an AI tool to look at the mentioning PRs and filter out false positives. Just maybe. We could do it without the filtering first and then see how many false positives we get. Yeah I'll probably give this a try, just not sure I'll have time before Christmas.

**Comment by gsmet** [comment]
Yeah the idea with this tooling is to do continuous improvements when we have time (except if they are completely broken, obviously).
